src/ig/files/_updatePublisher.sh

```diff
@@ -31,15 +45,17 @@ else
 	fi
 fi
 
-if "$upgrade"; then
-	message="Overwrite $jarlocation? (Y/N) "
-else
-	echo Will place publisher jar here: "$jarlocation"
-	message="Ok? (Y/N) "
+if [[ "$FORCE" != true ]]; then
+  if "$upgrade"; then
+    message="Overwrite $jarlocation? (Y/N) "
+  else
+    echo Will place publisher jar here: "$jarlocation"
+    message="Ok? (Y/N) "
+  fi
+  read -r -p "$message" response
 fi
 
-read -r -p "$message" response
-if [[ "$response" =~ ^([yY])$ ]]; then
+if [[ "$FORCE" ]] || [[ "$response" =~ ^([yY])$ ]]; then
```

Comment:
Hi @peturv! I was reviewing this pull request and noticed one thing that should be changed. I think the check for the force flag here should be checking `"$FORCE" == true`. As it is now, if a user is not using the `--force` flag and chooses `N` at the prompt, the download will still occur. Adding this line should fix this issue.

Reply:
Thanks! Feel a bit silly having missed that. Have posted an update.

Reply:
No worries! Thanks for pushing a fix!